# diff --git a/CMakeLists.txt b/CMakeLists.txt
# index b1eb4f5..e2581a2 100644
# --- a/CMakeLists.txt
# +++ b/CMakeLists.txt
# @@ -65,7 +65,7 @@ endif ()
#  set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
# 
#  if (WITH_SHARED_LIBUV)
# -  find_package(Libuv REQUIRED)
# +  find_package(LIBUV REQUIRED)
#    include_directories(${LIBUV_INCLUDE_DIR})
#  else (WITH_SHARED_LIBUV)
#    include_directories(deps/libuv/include)

# diff --git a/cmake/Modules/FindLibuv.cmake b/cmake/Modules/FindLibuv.cmake
# index 7b09848..7bad3db 100644
# --- a/cmake/Modules/FindLibuv.cmake
# +++ b/cmake/Modules/FindLibuv.cmake
# @@ -30,4 +30,4 @@ if(WIN32)
#  endif()
# 
#  INCLUDE(FindPackageHandleStandardArgs)
# -FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBUV DEFAULT_MSG LIBUV_LIBRARIES LIBUV_INCLUDE_DIR)
# +FIND_PACKAGE_HANDLE_STANDARD_ARGS(Libuv DEFAULT_MSG LIBUV_LIBRARIES LIBUV_INCLUDE_DIR)


--- a/cmake/Modules/FindLibuv.cmake
+++ b/cmake/Modules/FindLibuv.cmake
@@ -8,4 +8,4 @@
 FIND_LIBRARY(LIBUV_LIBRARIES NAMES uv libuv)
 
 INCLUDE(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(LIBUV DEFAULT_MSG LIBUV_LIBRARIES LIBUV_INCLUDE_DIR)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Libuv DEFAULT_MSG LIBUV_LIBRARIES LIBUV_INCLUDE_DIR)
